### ISSUE TITLE
Question To crypt data

### ISSUE THREAD

**Issue by sailsbox** [balderdashy/sails-mysql#188]
Hello,

I would like modifie sails adapter, to add a new fonction.
I would like crypt my data in my database, and use AES 256 bits.

Can-you help me ?

**Comment by vijayrajasekaran** [comment]
Try using bcrypt or bcryptjs.

**Comment by sailsbot** [comment]
Thanks for posting, @sailsbox.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
